src/api/Controllers/UploadController.cs

```diff
@@ -328,6 +345,17 @@ private void ValidateLithologyImports(List<int> importIds, List<LithologyImport>
             {
                 ModelState.AddModelError($"Row{lithology.index}", $"Borehole with {nameof(LithologyImport.ImportId)} '{lithology.value.ImportId}' not found.");
             }
+
+            // Check if all multi code list values are numbers
+            try
+            {
+                var codeListIdStrings = lithology.value.Color?.Split(",").Concat(lithology.value.OrganicComponent.Split(",")).Concat(lithology.value.GrainShape.Split(",")).Concat(lithology.value.GrainGranularity.Split(",")).Concat(lithology.value.Uscs3.Split(",")).Concat(lithology.value.Debris.Split(",")).ToList();
+                var codeListIds = codeListIdStrings?.Where(s => !string.IsNullOrEmpty(s)).Select(int.Parse).ToList() ?? new List<int>();
```

Comment:
Q: Lässt sich diese Logik in eine eigene Methode auslagern, da wir exakt dieselbe Logik an einer anderne Stelle haben? Oder ist das Teil des geplanten Refactorings?

Reply:
Ok. Ja das macht Sinn mit der Methode. Auch schon jetzt, vor dem refactoring.